liteflow-labs/liteflow-js
# Add example app

### Description

Moving https://github.com/liteflow-labs/minimal-app into the `liteflow-js` repo
Most libraries include an example app within their main repo, like [wagmi](https://github.com/wagmi-dev/wagmi/tree/main/examples) or [stripe](https://github.com/stripe/stripe-js/tree/master/examples).
This also ensures that the minimal-app / example is well maintained. Currently, it is way behind in terms of versioning. 

The GitHub action has been added to check that any modification in the library does not break the build of a default application.

Note 1: This example application is not deployed live as it doesn't provide much value other than testing
Note 2: Once this is merged, we can delete the https://github.com/liteflow-labs/minimal-app repository

### How to test

- `npm i` at the root
- `npm run build` to build the dependencies (hook package)
- `npm i` from the `example` directory
- `npm run dev` from the example directory
- Check that the application is working as expected

### TODO once merge:
- [ ] Remove / redirect minimal-app to this app
